packages/app-project/src/helpers/fetchTranslations/fetchTranslations.spec.js

```diff
@@ -75,11 +70,22 @@ describe('helpers > fetchTranslations', function () {
     expect(translation?.strings?.content).to.equal('test project page')
   })
 
+  it('should handle language codes that have partial capitalization', async function () {
+    const translation = await fetchTranslations({
+      translated_id: 1234,
+      translated_type: 'project',
+      language: 'zh-CN',
+      fallback: 'en',
```

Comment:
Add a new request matcher that expects the requested language to be `zh-cn,en` for this test.